### PR TITLE
feat: consume latest vsi with additional variables

### PR DESCRIPTION
**Pull request by rajatagarwal-ibm**
### Description

Consumes latest VSI module with additional variables

#### Changes that affect the core Terraform module or submodules
- [ ] Bug fix
- [ ] New feature
- [ ] Dependency update

#### Changes that don't affect the core Terraform module or submodules
- [ ] Examples or tests (addition or updates of examples or tests)
- [ ] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other

#### Release required?
Identify the type of release. For information about the changes in a semantic versioning release, see [Release versioning](https://terraform-ibm-modules.github.io/documentation/#/versioning).

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

If a release is required, replace this text with information that users need to know about the release. Write the release notes to help users understand the changes, and include information about how to update from the previous version.

Your notes help the merger write the commit message for the PR that is published in the release notes for the module.

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -276,6 +276,8 @@ variable "vsi" {
       image_name                      = string
       machine_type                    = string
       vsi_per_subnet                  = number
+      kms_encryption_enabled          = optional(bool)
```
Comment (toddgiguere) [inline variables.tf:279]: These should be defaulted or override values from the pattern.

**Review by toddgiguere — CHANGES_REQUESTED** [review]

**Comment by rajatagarwal-ibm** [comment]
Ran patterns/vsi and the boot volume is encrypted on the following settings.

```
kms_encryption_enabled        = true
skip_iam_authorization_policy = true
```

image

**Review by vburckhardt — APPROVED** [review]
lgtm

**Review by ocofaigh — APPROVED** [review]

**Review by toddgiguere — APPROVED** [review]

**Pull request merged**